papis/defaults.py

```diff
@@ -104,7 +104,21 @@ def get_default_opener() -> str:
                                    "month:int",
                                    "files:list",
                                    "notes:str",
-                                   "author_list:list"],
+                                   "author_list:list",
+                                   "tags:list",
+                                   "doi:str",
+                                   "ref:str",
+                                   "isbn:str",
+                                   "related_to:list",
+                                   "author:str",
+                                   "journal:str",
+                                   "note:str",
+                                   "number:int",
+                                   "volume:int",
+                                   "type:str",
+                                   "publisher:str",
+                                   "title:str",
+                                   "shorttitle:str"],
```

Comment:
I don't know if all of these are fair. Maybe we can leave more of them up to the user? From a quick look:
* `tags` are mostly strings, I think? Maybe we can leave this to `papis tag`?
* `related_to`: is this a custom key?
* `number`: I sometimes use this to mean `report number` for some techreports from research institutes. I'm not sure if BibTeX says this is an int?
* `volume`: I have some volumes that use roman numerals?

Reply:
That is fair, I can pare down this list a bit. I think the `tags` (as well as `related_to`) are leftovers from other changes meant to happen, but we can discuss this in future PRs. And good points with the other two, I'll remove them!